documentation/docs/reference/services/Profile.md

```diff
@@ -147,4 +167,87 @@ Response format:
         "python"
     ]
 }
-```
\ No newline at end of file
+```
+
+GET /profile/leaderboard/?limit=
+-------------------------
+
+Returns a list of profiles sorted by points descending. If a ``limit`` parameter is provided, it will return the first ``limit`` profiles. Otherwise, it will return all of the profiles.
+
+Response format:
+```
+{
+    profiles: [
+        {
+            "id": "github0000001"
+            "name": "John Smith",
+            "email": "john@gmail.com",
+            "github": "JSmith",
+            "linkedin": "john-smith",
+            "interests": [
+                "deep learning",
+                "ice skating",
+                "python"
+            ],
+            "points": 1000,
+        },
```

Comment:
Might want to only return the points and name in the payload since we dont want to expose emails. We could also use discord usernames if we want to provide more anonymity since displaying names could be kinda awk too

Reply:
Actually, good catch here. Looks like the docs here are using the wrong data fields. Let me correct this for this PR and work on the other changes in a new PR for granularity.